### PR TITLE
noiseprofile for Canon IXY 220F and Pentax ist D5

### LINKED ISSUE #15119: Noise Profiles - Pentax *ist DS
For noise profiles, please also attach the entire archive with `.pdf` plots.

[darktable-noiseprofile-20230709.tar.gz](https://github.com/darktable-org/darktable/files/12448638/darktable-noiseprofile-20230709.tar.gz)

### LINKED ISSUE #15598: Noise Profile for Canon IXY 220F
I have generated a noise profile for my Canon IXY 220F for inclusion upstream. Please see attached. Thank you :)

These were taken with CHDK v1.6.1 in order to take DNG raw images. Additionally, the PowerShot ELPH 110 HS and IXUS 125 HS should be the same camera.

~~[darktable-noiseprofile-20231107.tar.gz](https://github.com/darktable-org/darktable/files/13292111/darktable-noiseprofile-20231107.tar.gz)~~

Please see updated profile below.

### PR DESCRIPTION
**Pull request by rawfiner**
Fixes #15598
Fixes #15119

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -23,6 +23,19 @@
             {"name": "EOS-1Ds iso 1250", "iso": 1250, "a": [7.95278526496704e-05, 3.32459647343708e-05, 5.14955370294715e-05], "b": [5.09601367196328e-07, 5.3070173385365e-07, 6.62837426268686e-07]}
           ]
         },
+        {
+          "comment": "ixy 220f contributed by Chase Patterson",
+          "model": "Canon IXY 220F",
```
Comment (kmilos) [inline data/noiseprofiles.json:28]:
```suggestion
          "model": "IXY 220F",
```

**Comment by kmilos** [comment]
Depends on https://github.com/darktable-org/rawspeed/pull/565

File: data/noiseprofiles.json (hunk 1 above)
Comment (rawfiner) [reply data/noiseprofiles.json:28]: Thanks!

**Review by TurboGit — APPROVED** [review]
Thanks!

File: data/noiseprofiles.json (hunk 1 above)
Comment (chapatt) [reply data/noiseprofiles.json:28]: I'm curious why this was changed. The camera model is recognized as `Canon IXY 220F`. With the change, this profile can no longer be applied to images from the camera.

Here is a sample raw image if you'd like to reproduce (https://raw.pixls.us/getfile.php/6984/nice/Canon%20-%20Canon%20IXY%20220F%20-%2012bit%20CHDK%20ver.%201.6.1%20(4:3).DNG).

See the metadata panel from darktable with this image:

Comment (rawfiner) [reply data/noiseprofiles.json:28]: I will let @kmilos confirm, but once darktable-org/rawspeed#565 will be merged, the name will be IXY 220F.

Comment (kmilos) [reply data/noiseprofiles.json:28]: Correct. The change has now been merged, but not yet propagated to rawspeed stable branch nor the corresponding dt submodule yet (should happen for the next release).

In the meantime, one can easily add the lines to `cameras.xml` as in the PR.

Comment (chapatt) [reply data/noiseprofiles.json:28]: Ah, I see it adds an alias. Thank you!

Comment (kmilos) [reply data/noiseprofiles.json:28]: @chapatt This is now be working on dt master branch (need to update submodules of course) and nightly builds.

Comment (chapatt) [reply data/noiseprofiles.json:28]: Awesome! Thanks all!

**Pull request merged**